.github/workflows/backend-lint.yml

```diff
@@ -0,0 +1,31 @@
+name: Backend Lint
+
+on:
+  push:
+    branches: ["dev"]
+    paths: ["backend/"]
+  pull_request:
+    branches: ["dev"]
+
+defaults:
+  run:
+    working-directory: "backend/"
+
+jobs:
+  run-backend-lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20.x]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - name: Install dependencies
+        run: npm i
+      - name: Lint with Eslint
```

Comment:
**nitpick:** `Run linters` would make more sense since we may be running more than just ESLint.